tests/test_jinja.py

```diff
@@ -61,6 +63,207 @@ def test_jinja_platform(monkeypatch):
+
+
+def test_jinja_osx(monkeypatch):
+    template = "{{ osx }}"
+
+    monkeypatch.setattr(sys, 'platform', 'linux')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(sys, 'platform', 'darwin')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+
+def test_jinja_unix(monkeypatch):
+    template = "{{ unix }}"
+
+    monkeypatch.setattr(sys, 'platform', 'linux')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(sys, 'platform', 'darwin')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+
+def test_jinja_win(monkeypatch):
+    template = "{{ win }}"
+
+    monkeypatch.setattr(sys, 'platform', 'linux')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+    monkeypatch.setattr(sys, 'platform', 'darwin')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+
+def test_jinja_win32(monkeypatch):
+    template = "{{ win32 }}"
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+
+    monkeypatch.setattr(platform, 'architecture', lambda: ('32bit', ''))
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+    monkeypatch.setattr(platform, 'architecture', lambda: ('64bit', ''))
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+
+def test_jinja_win64(monkeypatch):
+    template = "{{ win64 }}"
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+
+    monkeypatch.setattr(platform, 'architecture', lambda: ('32bit', ''))
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(platform, 'architecture', lambda: ('64bit', ''))
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+
+def test_preprocess_selector_in_line():
+    line = "  - ccache    # [linux or osx]"
+    expected = "{{% if linux or osx %}}{0}{{% endif %}}".format(line)
+    assert preprocess_selector_in_line(line) == expected
+
+    line = "  - clcache    # [ win ]"
+    expected = "{{% if win %}}{0}{{% endif %}}".format(line)
+    assert preprocess_selector_in_line(line) == expected
+
+    line = "  - boost"
+    expected = line
+    assert preprocess_selector_in_line(line) == expected
+
+    line = "  - cmake  # cmake is a required dependency"
+    expected = line
+    assert preprocess_selector_in_line(line) == expected
+
+    line = "  - cmake  # [linux] cmake is a required dependency in linux"
+    expected = "{{% if linux %}}{0}{{% endif %}}".format(line)
```

Comment:
Shouldn't the comment (`cmake is a required dependency in linux`) be included into the expected line?